admin/tool/policy/classes/api.php

```diff
@@ -536,9 +570,9 @@ public static function accept_policies($policyversionid, $userid = null, $note =
         list($sql, $params) = $DB->get_in_or_equal($policyversionid, SQL_PARAMS_NAMED);
         $sql = "SELECT v.id AS versionid, a.*
                   FROM {tool_policy_versions} v
-                  LEFT JOIN {tool_policy_acceptances} a ON a.userid = :userid AND a.policyversionid = v.id
+                  LEFT JOIN {tool_policy_acceptances} a ON a.userid = :userid AND a.language = :language AND a.policyversionid = v.id
```

Comment:
This addition of language looks suspicious. I always considered is quite informative / extra info field. Not something you would check when JOINing tables. Also, does not it open a space for situation where there could exist two acceptance records, one for each language?

Reply:
That's exactly the reason why I've added the language here: to insert/update more than one row in the policy_acceptances table depending on the language. Without this change, only one row per policyversionid and userid was allowed, so there was no possibility to add an extra row if the language was different.

Reply:
But that is not right. We can't get to situation that a user agrees with the policy in English, but disagrees with the same policy in Spanish. That is why we have the unique index uq_versionuser defined in the table tool_policy_acceptances.